src/components/I18nOrdering.tsx

```diff
@@ -0,0 +1,18 @@
+import { Children, ReactNode, cloneElement, ReactElement } from "react";
+import { locales } from "src/locales";
+import { i18n } from "@lingui/core";
+
+// Component that inverts table columns if the current language is read right to left
```

Comment:
hmmm. what kind of downstream styling issues might we have if we invert table columns...?
wdyt @ohmygirth

Reply:
I will reply on Discord

Reply:
@ohmygirth Thoughts?

Reply:
@mrunbanked is this okay?